xclim/indicators/atmos/_temperature.py

```diff
@@ -1445,3 +1454,40 @@ def cfcheck(self, tas, snd=None):
     compute=indices.hardiness_zones,
     parameters={"method": "usda"},
 )
+
+chill_portions = TempHourly(
+    title="Chill portions",
+    identifier="cp",
+    units="",
+    standard_name="chill_portions",
+    # TODO: check what this does
+    cell_methods="",
+    description="Chill portions are a measure to estimate the bud breaking potential of different crops. "
+    "The constants and functions are taken from Luedeling et al. (2009) which formalises "
+    "the method described in Fishman et al. (1987). "
+    "The model computes the accumulation of cold temperatures in a two-step process. "
+    "First, cold temperatures contribute to an intermediate product that is transformed to a chill portion "
+    "once it exceeds a certain concentration. The intermediate product can be broken down at higher temperatures "
+    "but the final product is stable even at higher temperature. "
+    "Thus the dynamic model is more accurate than other chill models like the Chilling hours or Utah model, "
+    "especially in moderate climates like Israel, California or Spain.",
+    long_name="Chill portions after the Dynamic Model",
+    allowed_periods=["A"],
+    varname="cp",
+    compute=indices.chill_portions,
+)
+
+chill_units = TempHourly(
```

Comment:
Yes it very much sounds like what I want. The only question I have left it how or whether it can handle seasons that span across new years. E.g. december-februar/ growing season in the southern hemisphere. I imagine that the indexer can be adjusted to include these seasons but the resampling would combine the wrong months together? 

We do a hack where we shift the days so that a season always falls in the same year not sure whether this would make sense to do here. Its a little easier for us because we always have a 360_day calendar.